.goreleaser.yml

```diff
@@ -12,8 +12,9 @@ brews:
   folder: HomebrewFormula
   install: bin.install "replicated"
 universal_binaries:
-- replace: true
-  id: replicated
+- ids:
+  - cli
```

Comment:
This will break the install. Putting `cli`, names the binary also `cli`, but `bin.install` points to `replicated`.
But if you put `replicated` under `ids`, it doesn't create a `darwin_all` artifact.